lib/binary/ProofTraceParser.cpp

```diff
@@ -0,0 +1,97 @@
+#include <kllvm/binary/ProofTraceParser.h>
+
+namespace kllvm {
+
+#define INDENT_SIZE 2u
```

Comment:
```suggestion
constexpr auto indent_size = 2u;
```
+ the downstream name changes to references to `INDENT_SIZE`